tests/helper.py

```diff
@@ -101,3 +102,31 @@ def assert_meta_col_conversion(
         if len(record) != 0:
             fail_info = "Warnings raised when expected no warning on these conversions"
             pytest.fail(fail_info)
+
+
+def get_meta(ff: str, additional_params: dict):
```

Comment:
Easier than having to specify an empty dict as the second arg every time? get_meta("blah")
```suggestion
def get_meta(ff: str, additional_params: Optional[dict] = None):
    if additional_params is None:
        additional_params = {}
```